client/vscode/src/sampleCommandActivation.ts

```diff
@@ -14,3 +18,59 @@ export function logCommand(languageClient: LanguageClient) {
         languageClient.info(`Client: The log command has been executed`)
     }
 }
+
+export function qNetCommand(languageClient: LanguageClient) {
+    return async () => {
+        const request = {
+            command: 'aws/qNetTransform/startTransform',
```

Comment:
Why is this namespaced `qNetTransform`? Is this materially different than other transformations in shape or intent?

Reply:
that is the official feature name we get from PM, there's another java transform but probably not using this LSP.

Reply:
The Java transform will be refactored to use LSP as well so I fear there will be a whole lot of duplication if we make every server specific to a single development environment and programming language.

Are there any .NET specific cases that can be moved to a specific transformer strategy while keeping the core generic?